projects/exui-common-lib/src/lib/components/search-location/search-location.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { Observable, of } from 'rxjs';
+import { map, mergeMap } from 'rxjs/operators';
+import { LocationModel } from '../../models/location.model';
+import { LocationService } from '../../services/locations/location.service';
+
+@Component({
+  selector: 'exui-search-location',
+  templateUrl: './search-location.component.html',
+  styleUrls: ['./search-location.component.scss']
+})
+
+export class SearchLocationComponent implements OnInit {
+  @Input() public serviceId: string = '';
+  @Input() public locationType: string = '';
+  @Input() public disabled?: boolean = null;
```

Comment:
Just question, what is disabled for?

Reply:
This is if the conrtol is involved in page logic where it will need to be disabled. We are attempting to allow this to be used by working allocation they found it necessary to had this for find person.

Reply:
In workallocation they disable the search box in find person if they have other search option, I don't think we need to disable the search box in our case, this can be removed as our search box will be enabled all the time.

Reply:
I have spoke to work allocations and they said they need this in. This component is only about us.